angular.json

```diff
@@ -59,7 +59,8 @@
               "namedChunks": false,
               "extractLicenses": true,
               "vendorChunk": false,
-              "buildOptimizer": true
+              "buildOptimizer": true,
+              "aot": true
```

Comment:
This is the default as of angular 9

```suggestion
              "buildOptimizer": true
```

Reply:
Sure but
![image](https://user-images.githubusercontent.com/20106115/128503121-7b83c146-c0c0-4f79-a292-f0e97d321092.png)

Reply:
OK, weird... This is obviously a bug they have...